.golangci.yml

```diff
@@ -97,6 +97,10 @@ linters-settings:
     checks:
       - all
       - -ST1001 # ST1001 - Dot imports are discouraged
+  govet:
+    # Enable analyzers in addition to defaults.
+    enable:
+      - fieldalignment
```

Comment:
Where can one find the output of this linting step?
As mentioned in https://github.com/elastic/go-libaudit/pull/118#issuecomment-1184248805 at the moment there should be some output, as there are non aligned structs in use.

Reply:
The golangci-lint action adds PR annotations only and is configured to only report issues added in the change, so there will be none here. If you run it locally, you will see the complaints.